src/sentry/demo/apps.py

```diff
@@ -0,0 +1,11 @@
+from django.apps import AppConfig
+
+
+from .models import DemoOrganization, DemoUser
+
+
+class Config(AppConfig):
+    name = "sentry.demo"
+
+    def get_models(self):
+        return [DemoOrganization, DemoUser]
```

Comment:
I don't think we have to list these here, Django should be smart enough to pick them up from the `models.py` file.

Reply:
@wedamija good call! Just removed it.